FFmpegInterop/Source/FFmpegInteropMSS.cpp

```diff
@@ -399,6 +409,14 @@ HRESULT FFmpegInteropMSS::InitFFmpegContext(bool forceAudioDecode, bool forceVid
 
 				if (SUCCEEDED(hr))
 				{
+					// enable multi threading
+					unsigned threads = std::thread::hardware_concurrency();
+					if (threads > 0)
+					{
+						avVideoCodecCtx->thread_count = threads;
+						avVideoCodecCtx->thread_type = FF_THREAD_FRAME || FF_THREAD_SLICE;

```

Comment:
This should be a bitwise or instead of a logical or. `FF_THREAD_FRAME | FF_THREAD_SLICE`

Reply:
Absolutely. Fixed now.